.gitignore

```diff
@@ -40,3 +40,4 @@ coverage.txt
 .\#*
 
 .scootdata/*
+*/.scootdata/*
```

Comment:
does this do anything?

Reply:
yeah - running this test creates a tests/.scootdata dir that we don't want in here